### PR TITLE
WIP: Clean Up Bounty Creation Flow (#429)

### PR DESCRIPTION
**Pull request by olafghanizadeh**
##### Description
PR related to issue and bounty #429
##### Checklist

- [ -] linter status: 100% pass
- [- ] changes don't break existing behavior
- [- ] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
* UI

Thanks to @thelostone-mc and @eswarasai for help with Git issues :)

### CODE REVIEW

File: app/assets/v2/css/base.css [hunk 1]
```diff
@@ -460,26 +460,16 @@ input[type=text].loading {
     background-size: 30px;
 }
 
-.blocking_error {
+/* .blocking_error {
```
Comment (thelostone-mc) [inline app/assets/v2/css/base.css]: Remove this bit ?

File: app/dashboard/templates/submit_bounty.html [hunk 2]
```diff
@@ -127,36 +101,57 @@ <h3>Fund an Issue</h3>
...
+                                    {% include 'shared/metamask_estimate.html' %}
+                                    <div class="form__footer form__footer--right">
+                                        <button class="button button--primary btn-block" type="submit" id="submitBounty">Fund Issue</button>
+                                        <a target=new href="https://github.com/gitcoinco/gitcoinco/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp">Having trouble?  Click here.</a>
+                                    </div>
+                                </div>
+                            </form>
                         </div>
-                    </div>
+                    </form>
```
Comment (mbeacom) [inline app/dashboard/templates/submit_bounty.html]: Can we eliminate the duplicate closing tag for `<form>` ?

**Comment by owocki** [comment]
@olafghanizadeh ... sorry to hear that.. i wonder what big feature got merged to cause so many merge conflicts..

**Comment by thelostone-mc** [comment]
@owocki the translation support :P

**Comment by thelostone-mc** [comment]
@olafghanizadeh looks like the gitcoin bot kicked you out 😅  Wanna click on start work again ?

**Comment by owocki** [comment]
closing this as its been abandoned

**Pull request closed without merging**